src/transformers/visitors/checks/unsupportedVisitor.ts

```diff
@@ -79,6 +79,27 @@ export default {
     },
   },
 
+  BinaryOperation: {
+    enter(node: any) {
+      if (node.operator == "&&" || node.operator == "||") {
+        if (node.leftExpression.nodeType == "BinaryOperation" ) {
+          if ((node.leftExpression.operator == "&&" || node.leftExpression.operator == "||") && node.leftExpression.operator !== node.operator) {
+            throw new ParseError(
+              `A condition contains nested logical operators without brackets. Starlight does not know whether to read from right or left.`
+            );
+          }
+        }
+        if (node.rightExpression.nodeType == "BinaryOperation" ) {
+          if ((node.rightExpression.operator == "&&" || node.rightExpression.operator == "||") && node.rightExpression.operator !== node.operator) {
+            throw new ParseError(
+              `A condition contains nested logical operators without brackets. Starlight does not know whether to read from right or left.`
+            );
```

Comment:
Could you please change this error to say something like this ' Please use brackets to clarify the logic structure. A condition contains nested logical operators without brackets, which can lead to ambiguity in reading the logic flow.'